app/src/main/java/org/jellyfin/mobile/model/sql/entity/ServerEntity.kt

```diff
@@ -0,0 +1,24 @@
+package org.jellyfin.mobile.model.sql.entity
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.HOSTNAME
+import org.jellyfin.mobile.model.sql.entity.ServerEntity.Key.TABLE_NAME
+
+@Entity(tableName = TABLE_NAME, primaryKeys = [HOSTNAME])
+data class ServerEntity(
```

Comment:
We should use a unique identifier for the server so we can reference it from other tables in the future (like stored users). Should **not** be the server_id from the public server info.